package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": {
+    "axios": "^1.2.2",
```

Comment:
It seems `mocha` should be a dev dependency (Otherwise when you run `npm i` followed by `npm run test` it error's out saying it doesn't know what `mocha` is)